app/Http/Repository/UserRepository.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Suyabay\Http\Repository;
+
+use Suyabay\User;
+
+class UserRepository 
+{
+	
+    /**
+    * Return all user from the database
+    */
+	public function getAllUser()
+	{ 
+		return User::all();
+	}
+
+    /**
+    * Return all online users 
+    */
+	public function getOnlineUsers()
+	{ 
+		return $this->getAllUser()->where('active', 1);
+	}
```

Comment:
Your comments are out of place, they should start from the same line the `public` starts from.